src/utils/dmg.c

```diff
@@ -904,6 +979,14 @@ query obj layout options: \n\
   --pool=UUID	pool uuid\n\
   --cont=UUID	container uuid\n\
   --oid=oid	object oid.\n");
+	printf("\
+sysquery options:\n\
+  --group=STR	pool server process group (\"%s\")\n\
```

Comment:
(style)  space prohibited between function name and open parenthesis '('
(style)  spaces required around that '=' (ctx:VxV)